dashboard/src/components/AppView/AppView.tsx

```diff
@@ -124,7 +131,7 @@ class AppView extends React.Component<IAppViewProps, IAppViewState> {
 
   public handleEvent(e: MessageEvent) {
     const msg = JSON.parse(e.data);
-    const resource: IResource = msg.object;
+    const resource: any = msg.object;
```

Comment:
Is there any other way of setting the type without setting any? i.e using type unions?